README.md

```diff
@@ -120,6 +121,7 @@ spec:
 sharedIPLoadBalancing:
   loadBalancerType: cilium-bgp
   bgpNodeSelector: cilium-bgp-peering=true
+  ipHolderSuffix: myclustername1
```

Comment:
This needs changes in https://github.com/linode/linode-cloud-controller-manager/blob/main/deploy/chart/templates/daemonset.yaml and https://github.com/linode/linode-cloud-controller-manager/blob/main/deploy/chart/values.yaml

Reply:
got it, fixing now